uk.co.compucorp.civicrm.tasksassignments/js/test/tasks-assignments/controllers/modal/modal-assignment.controller.spec.js

```diff
@@ -516,7 +541,7 @@ define([
         data: resolvedValues.data,
         defaultAssigneeOptions: resolvedValues.defaultAssigneeOptions,
         session: resolvedValues.session,
-        config: {},
+        config: resolvedValues.config,
```

Comment:
What is the purpose of this change?

Reply:
This allows passing a custom `config` object, which is one of the dependencies for the modal assignment. One of the tests takes advantage of this and sets the config object equal to `{ customFieldIds: { 'caseType.category': 100 } }` where 100 is the number of the custom field as defined in the mock data.

https://github.com/compucorp/civihr-tasks-assignments/pull/409/files#diff-8c02655fb75ef8db8e05a681600238d4R85